packages/fx-core/src/plugins/solution/fx-solution/v2/utils.ts

```diff
@@ -58,6 +59,23 @@ export function isAzureProject(azureSettings: AzureSolutionSettings | undefined)
   return azureSettings !== undefined && HostTypeOptionAzure.id === azureSettings.hostType;
 }
 
+export function hasAzureResourcesToProvision(
```

Comment:
Can we reuse this method?
![image](https://user-images.githubusercontent.com/1658418/162355814-f4b08af4-4ef2-4236-8153-34e38cc3874e.png)

Reply:
Thanks, will use this~